build.sbt

```diff
@@ -13,5 +13,14 @@ libraryDependencies ++= Seq(
   // more dependencies here
   "io.circe" %% "circe-generic-extras" % "0.12.2",
   "io.circe" %% "circe-parser" % "0.12.2",
-  "com.dripower" %% "play-circe" % "2812.0", // compatible with Play 2.8.x
+  "com.dripower" %% "play-circe" % "2812.0"
 )
+libraryDependencies ++= Seq(
+  "org.tpolecat" %% "doobie-core" % "1.0.0-RC1",
+  "org.tpolecat" %% "doobie-postgres" % "1.0.0-RC1"
+)
+libraryDependencies ++= Seq(evolutions, jdbc)
+libraryDependencies ++= Seq(
+  "org.scalatestplus.play" %% "scalatestplus-play" % "3.0.x" % "test"
+)
+libraryDependencies += "net.postgis" % "postgis-jdbc" % "2.3.0"
```

Comment:
it seems you are continuously adding to the mutable `libraryDependencies` variable. Just add all dependencies in a single `Seq`